ComponentScreens/Login.js

```diff
@@ -0,0 +1,164 @@
+import React, {Component} from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+  TextInput,
+  TouchableOpacity,
+  ImageBackground,
+  KeyboardAvoidingView,
+} from 'react-native';
+import bgCover from '../assets/bgimage.png';
+
+export default class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      mail: '',
+      password: '',
+    };
+  }
+
+  isValidFields = () => {
+    const {mail, password} = this.state;
+
+    const mailformat = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
```

Comment:
It can be a global constant because we don't want it to be created every time the function is invoked.